test/files/run/repl-out-dir.check

```diff
@@ -11,13 +11,17 @@ repl-out-dir-run.obj
         $eval$.class
         $eval.class
     $line2
+        $eval$$iw$$iw$.class
+        $eval$$iw$.class
```

Comment:
does this mean every repl line generates two additional classfiles? is there a drawback to this? (performance?)